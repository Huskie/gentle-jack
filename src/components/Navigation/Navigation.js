import React from 'react';
import styled from 'react-emotion';

const StyledWrapper = styled('nav')`
  display: none;
  font-size: 0.75rem;
  font-weight: 700;
  letter-spacing: 1px;
  text-transform: uppercase;
  ul {
    list-style: none;
    margin: 0;
    padding: 0;
  }
  li {
    cursor: pointer;
  }
`;

const Navigation = () => (
  <StyledWrapper>
    <ul>
      <li>About</li>
    </ul>
  </StyledWrapper>
);

Navigation.propTypes = {};

Navigation.defaultProps = {};

export { Navigation };
