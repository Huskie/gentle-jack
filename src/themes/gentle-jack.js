const theme = {
  borderRadius: '4px',
  colours: {
    alpha: '#2742b8',
    beta: '#222',
    gamma: '#06f',
    delta: '#fff',
    epsilon: '#1b6cff',
    zeta: '#ffd561',
    eta: '#4048ef',
  },
  gradients: {
    alpha: '#4048ef, #2742b8',
  },
  spacing: {
    componentMargin: '16px',
  },
  sizing: {
    maxWidth: '1080px',
  },
};

export { theme };
