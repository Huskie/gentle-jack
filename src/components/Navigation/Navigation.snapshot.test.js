import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from 'emotion-theming';

import { Navigation } from './Navigation';
import { theme } from '../../themes/gentle-jack';

describe('Navigation snapshots:', () => {
  it('component renders with defaults', () => {
    const tree = renderer.create(
      <ThemeProvider theme={theme}>
        <Navigation />
      </ThemeProvider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
