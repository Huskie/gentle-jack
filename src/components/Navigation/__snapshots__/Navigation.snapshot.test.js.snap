// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation snapshots: component renders with defaults 1`] = `
.emotion-0 {
  display: none;
  font-size: 0.75rem;
  font-weight: 700;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  text-transform: uppercase;
}

.emotion-0 ul {
  list-style: none;
  margin: 0;
  padding: 0;
}

.emotion-0 li {
  cursor: pointer;
}

<nav
  className="emotion-0"
>
  <ul>
    <li>
      About
    </li>
  </ul>
</nav>
`;
